bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartservlet.xml

```diff
@@ -17,6 +17,9 @@
    <implementation class="org.eclipse.smarthome.ui.internal.chart.ChartServlet"/>
    
    <property name="service.pid" type="String" value="org.eclipse.smarthome.chart"/>
+   <property name="service.config.description.uri" type="String" value="ui:chart"/>
+   <property name="service.config.label" type="String" value="Chart UI"/>
```

Comment:
Hm, there isn't really such a thing as a "Chart UI" (compare to "Basic UI" etc.).

Reply:
"Chart" only ?